test/e2e/features/story_openshift.feature

```diff
@@ -36,31 +33,30 @@ Feature: 3 Openshift stories
 		Then with up to "4" retries with wait period of "1m" http response from "http://httpd-example-testproj.apps-crc.testing" has status code "200"
 		And executing "oc delete project testproj" succeeds
 
-	# Old: Local image to image-registry feature
+	# Local image to image-registry feature
 
-	@linux
-	Scenario: Create local image, push to registry, deploy
-		Given checking that CRC is running
-		And executing "podman pull quay.io/centos7/httpd-24-centos7" succeeds
+	@darwin @linux @windows
+	Scenario: Mirror image to OpenShift image registry
+		# mirror
 		When executing "oc new-project testproj-img" succeeds
-		And executing "podman login -u kubeadmin -p $(oc whoami -t) default-route-openshift-image-registry.apps-crc.testing --tls-verify=false" succeeds
-		And executing "podman tag quay.io/centos7/httpd-24-centos7 default-route-openshift-image-registry.apps-crc.testing/testproj-img/hello:test" succeeds
-		And executing "podman push default-route-openshift-image-registry.apps-crc.testing/testproj-img/hello:test --tls-verify=false" succeeds
-		And executing "oc new-app testproj-img/hello:test" succeeds
-		When executing "oc rollout status deployment hello" succeeds
+		And  executing "oc registry login --insecure=true" succeeds
```

Comment:
`oc registry login` and `podman login` both have different usecase. A user might build the image locally before pushing to internal registry and run it instead just pulling it from remote and pushing to internal registry (where `oc image mirror` is much suitable). I think we still need `podman login` test to make sure user can able to login to internal registry without any issue.

Reply:
yeah we can keep both scnearios...also relying on podman remote the local build now could be covered on all 3 platforms (need to run `crc podman-evn` for mac and windows)

Reply:
So to keep this here in e2e, the suggestion is that I clone the Scenario into two and one will be with `podman login` and the other with `oc registry login`? Using httpd-24 in both cases. Correct?

Reply:
yeah that will cover both scenarios  (+ oc podman-env) :) , which I think it is not covered at all either on e2e

Reply:
Hmmm, I returned the `podman login` scenario, but to use `podman-remote` I'll have to wrap all the steps because podman is called a different name on each of the platforms :(. Also, `eval $(podman-env)` is platform specific too.

Reply:
Done.